packages/prop-controllers/src/grid/grid.ts

```diff
@@ -144,7 +149,19 @@ export function getGridPropControllerElementChildren(
 function copyGridData(data: GridData, context: CopyContext): GridData {
   return {
     ...data,
-    elements: data.elements.map((element) => context.copyElement(element)),
+    elements: data.elements.flatMap((element) => {
```

Comment:
Here, similarly to https://github.com/makeswift/makeswift/pull/1019/commits/8647b90bf38197f24896bdbe5ef71d2dcd8ab135#r2078437155, I think using `filter` would make the intent clearer.